source/MaterialXView/Glsl/GLUtil/HardwarePlatform.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MATERIALX_HARDWAREPLATFORM_H
+
+// Platform macros. Only Windows platform is supported at this point
+// Linux and Mac flags well be enabled when fully supported.
+#if defined(_WIN64)
```

Comment:
"Master" controller to turn on/off platforms for building. Only Windows is turned on. Others end up with stub implementations as don't want to invoke pre-processor build errors.